lib/gherkin/parser.ex

```diff
@@ -34,7 +34,7 @@ defmodule Gherkin.Parser do
   end
   def process_lines(string) do
     {:ok, output} =
-      string |> String.split(~r/\r?\n/, trim: true)
```

Comment:
This was removing empty string lines which caused the line number to be off by however many there were in the original file. Since the line processor handles empty lines perfectly fine, I thought it was better to just process them and keep the line numbers in tact.